README.rst

```diff
@@ -26,9 +26,9 @@ You will be asked to select a serial port that you will use to connect with the
 The default baudrate is 38400 which should be correct for most BT modules.
 
 Both the port and the baud rate can be optionally provided on the command line if you prefer. For example,
-you can:
+you can::
```

Comment:
Hehehe. Yeah, that is a shortcut for starting a code block. Usually the indentation is 3 spaces, as it was before, but I think it does not really mind if it is 4.